test/specs/new/toplevel_paragraphs.md

```diff
@@ -13,13 +13,6 @@ text inside block code
 paragraph before hr
 * * *
 
-paragraph before head with hash
```

Comment:
Why is this part of the test removed?

Reply:
It was moved to another file with `pedantic: true` I'm not entirely sure why but it was like that in #1407